exporter/dorisexporter/config.go

```diff
@@ -95,3 +97,39 @@ func (cfg *Config) Validate() (err error) {
 
 	return err
 }
+
+const (
+	defaultStart = -2147483648 // IntMin
+)
+
+func (cfg *Config) start() int32 {
+	if cfg.HistoryDays == 0 {
+		return defaultStart
+	}
+	return -cfg.HistoryDays
+}
+
+func (cfg *Config) timeZone() (*time.Location, error) {
+	return time.LoadLocation(cfg.TimeZone)
+}
+
+const (
+	properties = `	
+PROPERTIES (	
+"replication_num" = "%d",	
+"enable_single_replica_compaction" = "true",	
+"compaction_policy" = "time_series",	
+"dynamic_partition.enable" = "true",	
+"dynamic_partition.create_history_partition" = "true",	
+"dynamic_partition.time_unit" = "DAY",	
+"dynamic_partition.start" = "%d",	
+"dynamic_partition.history_partition_num" = "%d",	
+"dynamic_partition.end" = "1",	
+"dynamic_partition.prefix" = "p"	
+)	
+`
+)
+
+func (cfg *Config) propertiesStr() string {
```

Comment:
```suggestion
func (cfg *Config) String() string {
```

to implement Stringer? https://go.dev/tour/methods/17

Reply:
never mind, seems like a very specific usage